### PR TITLE
Make culture deadblogs accessible

### PR DESCRIPTION
**Pull request by ioannakok**
## What does this change?
1. Makes culture deadblogs accessible - changes:
* ArticleTitle, ArticleMeta, SubMeta --> pillar 300
* KeyEvents below 980px --> pillar 400 / above 980px --> pillar 300
* CommentCount --> pillar 300

2. Fixes components accessiblity in deadblogs for the rest of the pillars according to the designs: https://www.figma.com/file/mjJRSYtNuyPi1NGYNOH2ZO/DCR-Live-blogs?node-id=2422%3A9088
3. Adds stories to capture the design requirements

## Why?
After discussion with @HarryFischer the design requirement is:
* When background is white, we use 400 for all pillars
* When background is gray we use 400 for News, Sports and Lifestyle, 300 for Culture and Opinion (Labs and SpecialReport are pending decision - for now they will be 300).

### Changes

| Before | After     |
|    :----:   |          ---: |
| Screenshot 2021-12-10 at 16 35 32    |  Screenshot 2021-12-10 at 17 04 14   |
| Screenshot 2021-12-10 at 16 35 46   |   Screenshot 2021-12-10 at 17 04 27     |
| Screenshot 2021-12-10 at 16 49 54   |   image   |
| Screenshot 2021-12-10 at 16 50 09   |   Screenshot 2021-12-10 at 17 05 12  |
| Screenshot 2021-12-10 at 16 35 04  |  Screenshot 2021-12-10 at 17 02 18  |

### CODE REVIEW

**Comment by ioannakok** [comment]
I am accepting a Chromatic diff for key events in dark mode. This PR changes some of them from 300 to 400. The rest remain 400. They should all be 500 and I will fix it in a new PR.

File: dotcom-rendering/fixtures/manual/articles.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		{ ...format, theme: ArticlePillar.Sport },
+		{ ...format, theme: ArticlePillar.Culture },
+		{ ...format, theme: ArticlePillar.Lifestyle },
+		{ ...format, theme: ArticlePillar.Opinion },
+		{ ...format, theme: ArticleSpecial.SpecialReport },
+		{ ...format, theme: ArticleSpecial.Labs },
+	];
+};
+
+export const getThemeFromFormat = (format: ArticleFormat): string => {
```
Comment (OllysCoding) [inline dotcom-rendering/fixtures/manual/articles.ts:24]: Is there perhaps a better name for this that highlights it returns the name of the theme as a string? (If I understood this correctly!)

File: dotcom-rendering/src/web/components/ArticleMeta.stories.tsx [hunk 2]
```diff
@@ -300,3 +304,32 @@ export const TwoContributorsStory = () => {
 	);
 };
 TwoContributorsStory.story = { name: 'Feature, with two contributors' };
+
+export const DeadBlogStory = () => {
+	return (
+		<>
+			{getAllThemes({
```
Comment (OllysCoding) [inline dotcom-rendering/src/web/components/ArticleMeta.stories.tsx:311]: I love this model for testing all themes work correctly :)

File: dotcom-rendering/fixtures/manual/articles.ts (hunk 1 above)
Comment (ioannakok) [reply dotcom-rendering/fixtures/manual/articles.ts:24]: Yes, this is what it does, thanks! 🙏 Changing the name, good call 🙂

**Comment by ioannakok** [comment]
[Make culture deadblogs accessible](https://trello.com/c/xj0LBbsC/452-make-culture-deadblogs-accessible)

**Review by DavidLawes — APPROVED** [review]
👍

**Pull request merged**